internal/config/config_test.go

```diff
@@ -377,3 +411,57 @@ func TestSeverity_MarshalText(t *testing.T) {
 func ptr[V any](v V) *V {
 	return &v
 }
+
+func TestParseExpirationDate(t *testing.T) {
+	tests := []struct {
+		name    string
+		date    string
+		want    ExpirationDate
+		wantErr error
+	}{
+		{
+			name:    "valid date",
+			date:    "2024/07/05",
+			want:    mustParseExpDate("2024/07/05"),
+			wantErr: nil,
+		},
+		{
+			name:    "invalid date",
+			date:    "2024-07-05",
+			want:    ExpirationDate{},
+			wantErr: ErrInvalidExpirationDate,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got, err := parseExpirationDate(tt.date)
+			if !errors.Is(err, tt.wantErr) {
+				t.Errorf("unexpected error: want: %v, got: %v", tt.wantErr, err)
+			}
+			if !got.Equal(tt.want.Time) {
+				t.Errorf("unexpected date: want: %v, got: %v", tt.want, got)
+			}
+		})
+	}
+}
+
+func TestExpirationDate_MarshalText(t *testing.T) {
+	date := mustParseExpDate("2024/07/05")
+	want := []byte("2024/07/05")
+
+	got, err := date.MarshalText()
+	if err != nil {
+		t.Errorf("unexpected error: want: %v, got: %v", nil, err)
+	}
+	if bytes.Equal(got, want) != true {
```

Comment:
Sorry, but I cannot accept this change :rofl: 

```suggestion
	if !bytes.Equal(got, want) {
```